python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -198,7 +198,7 @@ def set(self, value):
         def observe(self, value):
             pass
 
-        def inc(self):
+        def inc(self, amount=1):
```

Comment:
Can you add `*args, **kwargs` to all these too to prevent future issues?

Reply:
Sure, will do.

Reply:
Done.